src/styles/dictionary_concepts.scss

```diff
@@ -411,3 +411,31 @@ input:-webkit-autofill:active {
 input[type=text] {
   text-overflow: ellipsis;
 }
+
+.conceptDetails {
+  max-height: 80px;
+}
+.cielConceptsList {
+  border: 1px solid #bbb;
```

Comment:
How is this related to execute a search only when ENTER is pressed?

Reply:
Since the library(AsyncSelect) that was being used could not allow one to press enter to search I had to recreate my own code that creates a dropdown and allows selecting of a specific concept. That's why I included all the CSS code